Source/DataSources/LabelGraphics.js

```diff
@@ -318,7 +321,15 @@ define([
          * @memberof LabelGraphics.prototype
          * @type {Property}
          */
-        disableDepthTestDistance : createPropertyDescriptor('disableDepthTestDistance')
+        disableDepthTestDistance : createPropertyDescriptor('disableDepthTestDistance'),
+
+        /**
+         * Gets or sets the ability to modify text characters direction.
+         * @memberof LabelGraphics.prototype
+         * @type {Property}
+         * @default false
+         */
+        rightToLeft: createPropertyDescriptor('rightToLeft')
```

Comment:
This is basically an optimization, right?  If the string doesn't contain RTL characters it is a no-op?

Reply:
yes

Reply:
Since this is just an optimization, how do you feel about having a single static `Label.enableRightToLeftDetection` variable that an application sets once globally rather than having to set it on every label?  It will save memory when there are many labels and I doubt it will have a performance impact in RTL apps since I would expect most of the labels to be RTL in that case.  Thoughts?

Reply:
Sounds very interesting.
I'm not sure 100% how to implement it, but if it is the right way then I should do it as you offered

Reply:
after a little bit of thinking I understand how to implement it, it will clean a lot of the work and it is nice.
my question is: how to declare the static variable so each application could manage it?

Reply:
Just add a property directly on the `Label` object.  Something like this should work.

```javascript
   /**
     * Enabled detection and support of right to left text.
     *
     * @type {Boolean}
     * @default false
     *
     * @example
     * //Set once at the start of the application
     * Cesium.Label.enableRightToLeftDetection = true;
     * 
     * var myLabelEntity = viewer.entities.add({
     *   label: {
     *     id: 'my label',
     *     text: 'זה טקסט בעברית \n ועכשיו יורדים שורה'
     *   }
     * });
     */
    Label.enableRightToLeftDetection = false;
```

This will show up as static in the documentation and users just set `Cesium.Label.enableRightToLeftDetection = true` once in their app to use it. In your own code, you would just check `Label.enableRightToLeftDetection` directly as well.

Thanks.